start-alertmanager.sh

```diff
@@ -150,11 +153,12 @@ fi
 # Wait till Alertmanager is available
 RETRIES=5
 TRIES=0
-until $(curl --output /dev/null -f --silent http://localhost:$ALERTMANAGER_PORT) || [ $TRIES -eq $RETRIES ]; do
-	((TRIES = TRIES + 1))
-	sleep 5
-done
-
+if [ ! "$QUICK_STARTUP" = "1" ]; then
+    until $(curl --output /dev/null -f --silent http://localhost:$ALERTMANAGER_PORT) || [ $TRIES -eq $RETRIES ]; do
+    	((TRIES = TRIES + 1))
+    	sleep 5
```

Comment:
You could probably reduce this to 1 second and increase the retries count.

Reply:
BTW, curl has all the built-in functionality for retrying. No real need for this loop.